packages/stems/src/components/PillButton/PillButton.tsx

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames'
+
+import { Button, Type as ButtonType } from 'components/Button'
+
+import styles from './PillButton.module.css'
+import { PillButtonProps, Type as PillButtonType } from './types'
+
+const TYPE_STYLE_MAP = {
+  [PillButtonType.PRIMARY]: styles.primary,
+  [PillButtonType.SECONDARY]: styles.secondary
+}
+export const PillButton = (props: PillButtonProps) => {
+  const {
+    text,
+    onClick,
+    type = PillButtonType.PRIMARY,
+    className,
+    textClassName
+  } = props
+  return (
+    <Button
+      className={cn(styles.button, TYPE_STYLE_MAP[type], className)}
+      textClassName={cn(styles.buttonText, textClassName)}
+      type={ButtonType.COMMON}
```

Comment:
love that we can reuse `type` here. just a headsup, i am wanting us to move away from the `type` prop since it is overloaded with the actual button `type` prop which is `submit | button` etc. so to avoid having that with pill button, can we use `variant` and just pass that into button's `type` for now?

Reply:
also, we probably want to pass the rest of button props through yeah? so we get id, data-testid, etc. i like doing
```typescript
const {className, textClassName, ...other} = props
<Button
   className={...}
   textClassName={...}
   {...other}
/>